app/src/main/java/org/mozilla/focus/cookiebannerexception/CookieBannerExceptionDetailsSwitch.kt

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.cookiebannerexception
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import androidx.constraintlayout.widget.ConstraintLayout
+import org.mozilla.focus.R
+import org.mozilla.focus.databinding.SwitchWithDescriptionBinding
+
+class CookieBannerExceptionDetailsSwitch @JvmOverloads constructor(
```

Comment:
We may need to uplift this to AC as it looks like we ave a similar [behaviour of Fenix](https://github.com/mozilla-mobile/fenix/blob/main/app/src/main/java/org/mozilla/fenix/trackingprotection/SwitchWithDescription.kt)